politeiawww/database/database.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2018 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package database
```

Comment:
Everything in here is specific to cms and we may have other database interfaces in the future for other stuff.  Should we call this package and interface something like `cmsdatabase` instead?  I'm not sure.  What do you think?

Reply:
yah makes sense, do you want me to rename the dir as well?